vita3k/gui/include/gui/state.h

```diff
@@ -146,6 +157,9 @@ struct GuiState {
     gui::LiveAreaState live_area;
     gui::AppsSelector app_selector;
 
+    std::map<std::string, User> users;
+    std::map<std::string, ImGui_Texture> users_avatar;
```

Comment:
Any reason this isn't just a part of the User struct above?

```
struct User {
    std::string id;
    std::string name;
    std::string avatar;
    std::string theme_id;
    bool use_theme_bg;
    std::string start_type;
    std::string start_path;
    std::vector<std::string> backgrounds;
    ImGui_Texture avatar_texture; // +
};
```

Reply:
hmm i have no tested inside so hmm good thinks i look